README.md

```diff
@@ -242,4 +242,18 @@ to be used in production. The next steps we plan to work on are:
     forbidden, making it not Turing-complete.
 
 See [RATIONALE.md](./RATIONALE.md) for the design rationale and a more detailed
-comparison with a selection of these languages.
+comparison with these languages.
+
+### Summary
+
+| Language | Typing                        | Recursion  | Evaluation | Side-effects                                     |
+|----------|-------------------------------|------------|------------|--------------------------------------------------|
+| Nickel   | Gradual (dynamic + static)    | Yes        | Lazy       | Yes (constrained)                                |
```

Comment:
I think currently, the side-effects are a lie. Maybe we should change that to `Yes (constrained, planned)`?
That doesn't have much to do with this PR, though.

Reply:
Done in 3a283d2f1f85142c605ec3706ed72418cae60240